tests/test-valid-json/test.yaml

```diff
@@ -0,0 +1,19 @@
+requires:
+  min-version: 8
+
+args:
+  - -T
+
+pcap: false
+
+exit-code: 1
+
+checks:
+  # check that we have the following entries in eve.json
+  # match rule load failure reason as it is super long
+  - filter:
+      count: 2
+      match:
+        event_type: engine
+        engine.module: detect
+
```

Comment:
What is the valid json the name of the test refers to ?
Why does this rule fail to load ?

Reply:
The signature is so long that it triggers truncation of the log message which end up not being proper json because of that. For the signature it self, I think it is missing a semi coma but this is not critical for the test.

Reply:
Ok thanks